Functii/src/b/FahrenheitToCelsius.java

```diff
@@ -0,0 +1,21 @@
+package b;
+import java.util.Scanner;
+
+public class FahrenheitToCelsius {
+
+    public static void main(String[] args) {
+        System.out.println("Please introduce temperature in Fahrenheit");
+        Scanner sc = new Scanner(System.in);
+        double fahrenheit = sc.nextDouble();
+        convertFarToCel(fahrenheit);
+    }
+
+    public static double convertFarToCel(double fahrenheit) {
+        double celsius = ((fahrenheit - 32) * 5) / 9;
+        System.out.println("Your body temperature in Celsius degrees is " + celsius);
+        if (celsius > 37) {
```

Comment:
Functia care face conversia nu ar trebui sa faca si verificari asupra rezultatului, verificarea daca rezultatul conversiei e `>` ca 37 trebuia facuta in metoda main.